launcher.py

```diff
@@ -1,9 +1,15 @@
 from src.lib.bot import SCUFFBOT
-import argparse
+from dotenv import load_dotenv
+import yaml
+import os
 
-parser = argparse.ArgumentParser(description='Runs ScuffBot.')
-parser.add_argument('--dev', action='store_true', help='Enable development mode.')
-args = parser.parse_args()
+load_dotenv()
 
-bot = SCUFFBOT(args.dev)
-bot.run()
+with open(os.environ["APPLICATION_IDS"], "r", encoding="utf-8") as f:
```

Comment:
Do you think there would be a point having a check here to show a friendly message when this env var does not result in a file? Or do you think the `open()` call raising an exception as a result of `local-secrets` not existing is sufficient?

Reply:
Couldn't hurt

Reply:
Add to this PR? Or nah

Reply:
Probs in another PR - I think Scuffbot needs a lot of rework anyway lol